src/A2.py

```diff
@@ -137,7 +151,7 @@ def parse(self):
                                 continue
 
         return {
-            'file': self.file_name,
+            'file': self.file_name.replace("\\", "/"),
```

Comment:
Remove. The script is automatically run on unix machines. I guess you used it on windows to check diffs?